### PR TITLE
Planning: fix no planning output when a car stops behind adc

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

File: modules/planning/toolkits/deciders/front_vehicle.cc [hunk 1]
```diff
@@ -302,6 +302,7 @@ std::string FrontVehicle::FindPassableObstacle(
 
 void FrontVehicle::MakeStopDecision(ReferenceLineInfo* reference_line_info) {
   const auto& adc_sl = reference_line_info->AdcSlBoundary();
+  const auto& vehicle_sl = reference_line_info->VehicleSlBoundary();
```
Comment (lianglia-apollo) [inline modules/planning/toolkits/deciders/front_vehicle.cc]: Please put variable declaration as close as where you will use it.

File: modules/planning/common/reference_line_info.h [hunk 2]
```diff
@@ -203,6 +204,8 @@ class ReferenceLineInfo {
 
   SLBoundary adc_sl_boundary_;
 
+  SLBoundary vehicle_sl_boundary_;
```
Comment (lianglia-apollo) [inline modules/planning/common/reference_line_info.h]: what is the difference between adc_sl_boundary_ and vehicle_sl_boundary_?

Comment (JasonZhou404) [reply modules/planning/common/reference_line_info.h]: adc_sl_boundary is generated by the stitching point while the latter is generated by the realtime vehicle position

Comment (ycool) [reply modules/planning/common/reference_line_info.h]: can you add these comments for these two variables? thanks.

**Pull request merged**